packages/models/src/public_opinion/v2.rs

```diff
@@ -71,7 +71,7 @@ pub struct Institution {
 
 #[api_model(base = "/reviews/v1", table = reviews, iter_type=QueryResponse)]
 pub struct Review {
-    #[api_model(summary, primary_key, action = delete )]
+    #[api_model(summary, primary_key, action = delete, read_action = find_by_id )]
```

Comment:
You don't nee to add find_by_id.
action, action_by_id, read_action, query_action are designed to be used in the client side.

On the server side, automatically it generates `get_repository` to write data and `query_builder` for flexible query.